topo_processor/util/conversions.py

```diff
@@ -1,6 +1,6 @@
 import re
 from datetime import datetime
-from typing import Dict
+from typing import Any, Dict
```

Comment:
unused `Any' import?

Reply:
Good spot. Fixed in https://github.com/linz/topo-processor/pull/512